Compound.ipynb

```diff
@@ -429,6 +447,58 @@
     "df"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Determine the number of studies\n",
+    "Parse the result returned to determine the number of studies\n",
+    "Names like idrXXX/screenA and idrXXX/screenB are counted as one study."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "code_folding": [
+     1
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "curated = {\"Screen\":[]}\n",
+    "for s in pd.unique(df[[\"Screen\"]].values.ravel()):\n",
+    "    curated[\"Screen\"].append(s.split(\"/\")[0])\n",
+    "\n",
+    "df_studies = pd.DataFrame(curated)\n",
+    "studies = pd.unique(df_studies[[\"Screen\"]].values.ravel())\n",
+    "print(len(studies))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/markdown": [
+       "The resuls consists of 6 studies and 2099 images."
+      ],
+      "text/plain": [
+       "<IPython.core.display.Markdown object>"
+      ]
+     },
+     "execution_count": 32,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from IPython.display import Markdown as md\n",
+    "md(\"The resuls consists of {} studies and {} images.\".format(len(studies), len(pd.unique(df[[\"Image\"]].values.ravel()))))"
```

Comment:
Suggested formulation:
``The result cosists of X images in Z studies.``
(sorry, do not want to susggest in a commit, as can easily make a mistake in the syntax.